src/main/java/seedu/address/storage/JsonAdaptedMeeting.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.storage;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Meeting;
+
+import javax.xml.namespace.QName;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_TIME;
+
+/**
+ * Jackson-friendly version of {@link Meeting}.
+ */
+class JsonAdaptedMeeting {
```

Comment:
This class LGTM, but just wondering whether we need to add validation logic that checks whether start time is earlier than end time here? Although likely unnecessary.